kml/edit-kml-ground-overlay/README.md

```diff
@@ -15,12 +15,13 @@ Use the slider to adjust the opacity of the ground overlay.
 ## How it works
 
 1. Create an `Envelope` defining the geometry of the overlay.
-2. Create a `KmlIcon` using a  URI linking to an image.
+2. Create a `KmlIcon` using a `Uri` linking to an image.
```

Comment:
Since URI is an acronym and not a class in our API, we should keep it capitalized
```suggestion
2. Create a `KmlIcon` using a URI linking to an image.
```